blocks/filter-group/filter-group.css

```diff
@@ -0,0 +1,79 @@
+.filter-group-wrapper {
+  padding-right: 0 !important;
+
+  @media (min-width: 76rem) {
+    padding: 0 2rem !important;
+  }
+}
+
+.filter-group {
+  margin-block: 1.5rem;
+}
+
+.filter-group__list {
+  display: flex;
+  gap: 0.5rem;
+  overflow-x: auto;
+  padding-block-end: 1rem;
+  padding-inline-start: 0;
+  scrollbar-width: none; /* Firefox */
+  border-bottom: 1px solid var(--spectrum-gray-700);
+  transition: padding-inline-start 100ms ease-in-out;
+}
+
+/* Hide scrollbar for Chrome, Safari and Opera */
+.filter-group__list::-webkit-scrollbar {
+  display: none;
+}
+
+.filter-group__list:has(.filter-group__button:first-child:focus) {
+  padding-inline-start: 0.5rem;
+}
+
+.filter-group__label {
+  margin-block-end: 0.5rem;
+  color: var(--spectrum-gray-700);
+  font-size: 1rem;
+}
+
+.filter-group__button {
+  --background-color: light-dark(
```

Comment:
Heya remember to use the selectors so that the toggle works correctly for manual settings.

Reply:
I touched base with Osmond yesterday on this and verified our manual toggle settings are still working as expected with this function. But I'd like to get another pair of eyes to test this on their machine to make sure it's still working as expected.

Reply:
Thanks for updating me on the context!